pkg/common/constants.go

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright 2025 The Kubeflow Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package common
+
+const (
+	LABEL_ALIYUN_COM_GPU_COUNT = "aliyun.com/gpu-count"
+)
```

Comment:
I suggest combining all constants into a single const block for better readability and organization.